setup.py

```diff
@@ -136,79 +58,120 @@ def write_meta_yaml(filename='devtools/conda-recipe/meta.yaml'):
 #def read(fname):
 #    return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+
 def find_package_data(data_root, package_root):
     files = []
     for root, dirnames, filenames in os.walk(data_root):
         for fn in filenames:
             files.append(relpath(join(root, fn), package_root))
     return files
 
-def check_dependencies():
-    from distutils.version import StrictVersion
-    found_openmm = True
-    found_openmmtools = True
-    found_openmm_711_or_earlier = True
-    found_numpy = True
-
-    try:
-        from simtk import openmm
-        openmm_version = StrictVersion(openmm.Platform.getOpenMMVersion())
-        if openmm_version < StrictVersion('7.1.1'):
-            found_openmm_711_or_earlier = False
-    except ImportError as err:
-        found_openmm = False
 
-    try:
-        import numpy
-    except:
-        found_numpy = False
-
-    try:
-        import openmmtools
-    except:
-        found_openmmtools = False
-
-    msg = None
-    bar = ('-' * 70) + "\n" + ('-' * 70)
-    if found_openmm:
-        if not found_openmm_711_or_earlier:
-            msg = [bar, '[Unmet Dependency] BLUES requires OpenMM version 7.1.1. You have version %s.' % openmm_version, bar]
-    else:
-        msg = [bar, '[Unmet Dependency] BLUES requires the OpenMM python package. Please install with `conda install -c omnia openmm=7.1.1` ', bar]
-
-    if not found_numpy:
-        msg = [bar, '[Unmet Dependency] BLUES requires the numpy python package. Refer to <http://www.scipy.org/scipylib/download.html> for numpy installation instructions.', bar]
-
-    if not found_openmmtools:
-        msg = [bar, '[Unmet Dependency] BLUES requires the openmmtools python package. Please install with `conda install -c omnia openmmtools=0.14.0`', bar]
-
-    if msg is not None:
-        import textwrap
-        print()
-        print(os.linesep.join([line for e in msg for line in textwrap.wrap(e)]), file=sys.stderr)
-        #print('\n'.join(list(textwrap.wrap(e) for e in msg)))
+# def check_dependencies():
```

Comment:
Why a bunch of commented-out code here?

Reply:
This came from  #144 but I took it out as installing packages this way typically ended up in lots of version conflicts.

Reply:
Sorry, should have said that differently. Usually if you want to remove code, you would remove it. And if you want it, you keep it in. Commenting it out is sort of this strange middle ground where you have removed it only halfway. Why not go the whole way?